components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/restapi/publisher/ApisApiServiceImplUtils.java

```diff
@@ -70,9 +74,7 @@
 import java.net.URL;
 import java.net.UnknownHostException;
 import java.nio.charset.StandardCharsets;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Let's reset those wildcard imports

Reply:
Resolved in a05bcd7fc00248a0d25d1496cf127e5caee4cdfc